### DISCUSSION TITLE
should objectives still support multiple conditions

### DISCUSSION THREAD

**Discussion by deltanedas** [space-wizards/space-station-14#19918] [Technical]
iirc in ss13 lings or something have 2-part objectives but nothing uses it here

is there any reason to support it since it makes things a lot more complex with ecsified objectives and nothing currently uses it

**Comment by mirrorcult** [comment]
how does it concretely make things more complicated, not sure i understand how it would

**Reply by deltanedas** [reply]
have to check a list<entityuid> for all their different ObjectiveComponent etc and events
instead of just having it be a component on a single identity (soon) or a component on the objective (when objectives are entity)

@mirrorcult

**Reply by deltanedas** [reply]
ok i remove :trollface:
